corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -24,6 +24,12 @@ <h2>{% trans 'MultiMedia Sizes' %}</h2>
     <div id="multimedia-sizes-container-2">
       {% include 'app_manager/partials/multimedia_sizes.html' with version=other_app.version %}
     </div>
+    <div id="multimedia-sizes-diff">
```

Comment:
I think you could remove the two divs above. That'd make this section more like the files section, where on the diff page you only see changes, you don't see the original content from either build.

Reply:
hmm, i thought it was useful to show these here in case one wanted to see the full size, otherwise one would need to locate the other version in the releases list to see it's total media size. Plus It's also a good reminder that the update would need say A MBs of MM download even if the MM in versions don't differ exactly by A MBs.
What do you think?
I think i can align these better so they appear next to each other instead of under each other.

Reply:
Hmm...thought about it some more but I don't feel strongly either way, it's not a lot of content. Whichever you'd prefer.